pastis/temporal_analysis/close_loop_analysis.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+
+def req_closedloop_calc_recursive(Gcoro, Gsensor, E0coro, E0sensor, Dcoro, Dsensor, t_exp, flux, Q, Niter,dh_mask,norm):
+    """
+    Parameters:
+    ----------
+    Gcoro: numpy ndarray
+        sensitivity matrix calculated at the coronagraphic plane: list of electric fields gathered by
+        poking one segment with one mode at a time and subtracting the un-aberrated coronagraphic field.
+    Gsensor: numpy ndarray
+        sensitivity matrix calculated at the wavefront sensor plane: list of electric field at the wfs plane
+        gathered by poking one segment with one mode at a time ans subtracting the unaberrated electric field
+        at the wfs plane
+    E0coro: numpy ndarray
+        unaberrated coronagraphic electric field
+    E0sensor: numpy ndarray
+        reference un-aberrated electric field seen by wavefront sensor
+    Dcoro:
```

Comment:
there are a bunch of function variables undefined in the docstrings, please specify